ts/features/pn/aar/screen/SendAARCanEducationalScreen.tsx

```diff
@@ -0,0 +1,108 @@
+import { SafeAreaView } from "react-native-safe-area-context";
```

Comment:
also, when possible, please keep the file and variable naming `[...]Aar[...]` instead of `[...]AAR[...]`, so that camelCase and PascalCase names are more readable